### ISSUE TITLE
implement limit of invites

### ISSUE THREAD

**Issue by Darkool** [LB--/KataParty#2]
Hello, let me say that it's an amazing plugin, what about add a new feature (Shop invites) i mean player should have a limit of invites (then they can purchase new invites) or maybe by permissions?, regards

**Comment by LB--** [comment]
I'm thinking about implementing this in a similar way to the /kpback command's permission-based grace periods. So, in the config file you'd have something like this:

``` yaml
party-invite-limits:
  default: 4
  admin: -1
```

Then you would give the permission nodes `KataParty.invite.limit.default` or `KataParty.invite.limit.admin`.

There are plenty of plugins already that allow purchasing permission nodes, so you could use that for buying higher invite limits. By adjusting the config you could have as many tiers as you wanted.

**Comment by Darkool** [comment]
thats sound good, what about this?: 
party-invite-limits:
  default: 40   <--- money
  admin:  0  <--- money

Later GUI 
Click to purchase 1(default) invite 
*PARTIES.YML

 parties:
- ==: com.lb_stuff.kataparty.Party
  invite: true
  sticky: false
  visible: true
  pvp: false
  tp: true
  name: aero
  invites: 1 <--- number of invites

anyway I can also your idea ;)

**Comment by LB--** [comment]
No, you're asking me to make a massive change to the behavior of my plugin _just for your case_. There a plenty of servers that do not need or want invite purchasing.

**Comment by Darkool** [comment]
of course, im understand this, by this reason i said you that your idea is good, sorry if I didn't express properly
